Sources/Algorithms/Windows.swift

```diff
@@ -46,8 +46,11 @@ extension Collection {
 /// A collection wrapper that presents a sliding window over the elements of
 /// a collection.
 public struct Windows<Base: Collection> {
-  public let base: Base
-  public let size: Int
+  @usableFromInline
+  internal let base: Base
+  
+  @usableFromInline
+  internal let size: Int
```

Comment:
Ditto here: does `size` need to be made internal?

Reply:
@LucianoPAlmeida I take it the 👍 indicates that you'd agree with reverting this change?

Reply:
Ah sorry, I just thought that it was the same topic as the on going discussion above... that's why I just reacted and didn't respond here :)